src/core/dd.js

```
@@ -0,0 +1,63 @@
+    } else {
+      unknown.push(token);
+    }
+  }
+  if (unknown.length > 0) {
+    const msg = `dd: unrecognized argument '${unknown[0]}'\n`;
+    return { stdout: '', stderr: msg, env: Object.assign({}, env, {'?': 1}) };
+  }
+  const ifile = opts.if;
+  const ofile = opts.of;
+  const bs = opts.bs !== undefined ? Number(opts.bs) : 512;
+  const count = opts.count !== undefined ? Number(opts.count) : Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)